app/code/Magento/Tax/Model/Config.php

```diff
@@ -845,7 +845,7 @@ public function needPriceConversion($store = null)
                 case self::DISPLAY_TYPE_BOTH:
                     return self::PRICE_CONVERSION_MINUS;
                 case self::DISPLAY_TYPE_INCLUDING_TAX:
-                    $res = false;
+                    $res = true;
```

Comment:
Could you please provide more information about the changes because right now it looks weird that we should change result value to the opposite. Thank you!